docs/content/datasources.md

```diff
@@ -191,6 +192,37 @@ $ echo '{{ (ds "foo" "/second/p1").Value }}' | gomplate -d foo=aws+smp:///foo/
 aaa
 ```
 
+## Using `aws+asm` datasource
+
+### URL Considerations
+For `aws+asm`, only the _scheme_ and _path_ components are necessary to be defined. Other URL components are ignored.
+
+### Output
+
+The output will be a single `GetSecretValueOutput` object from the [AWS SDK for Go](https://docs.aws.amazon.com/sdk-for-go/api/service/secretsmanager/#GetSecretValueOutput)
```

Comment:
> The output will be a single `GetSecretValueOutput` object

This behaviour has been quite awkward with the `aws+smp` datasource, and I totally regret it 😉. Can we avoid this and instead just return the `SecretString` string field? That way it's easier to substitute other datasources with the same template (imagine being able to run the same template with either a `vault` datasource or an `aws+sm` datasource, or a YAML `file` datasource for testing)...